src/components/AppHeader/UserMenu.js

```diff
@@ -63,7 +69,7 @@ const UserMenu = ({ user }) => {
 					{user.name && user.name.charAt(0).toLocaleUpperCase()}
 				</Avatar>
 				&nbsp;&nbsp;
-				{user ? user.name : 'Loading...'}
+				<span className="userName">{user ? user.name : 'Loading...'}</span>
```

Comment:
`css` prop returns a unique className based on the styles with the help of `@emotion/hash`.
```suggestion
				<span className={userName}>{user ? user.name : 'Loading...'}</span>
```

Reply:
Missed that one, thanks! :)